### ISSUE TITLE
Slim::Engine: Option :escape_html is invalid

### ISSUE THREAD

**Issue by dmitriy-kiriyenko** [trailblazer/cells-slim#3]
During the tests run in my application I face:

```
Slim::Engine: Option :escape_html is invalid
Slim::Engine: Option :escape_attrs is invalid
```

Before I started using `cells-slim` but using just `slim-rails` in regular Rails views, there was no such warning.

[My Gemfile now](https://gist.github.com/dmitriy-kiriyenko/d65a0f75c464840bf3e9).

I'll of course try to look by myself, but everything can happen, and I can forget about it.

**Comment by dmitriy-kiriyenko** [comment]
That is, for sure, odd, because I don't see such an option anywhere in source code of `cells-slim`.

**Comment by leolukin** [comment]
I see the same warnings in my log

**Comment by porbas** [comment]
See: apotonick/cells#255

**Comment by apotonick** [comment]
This is fixed in cells-4.0.0.beta6 and the cells-slim-0.0.2.
